drop-in/src/main/java/com/adyen/checkout/dropin/internal/ui/model/DropInPaymentMethodInformation.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by ararat on 9/11/2023.
+ */
+
+package com.adyen.checkout.dropin.internal.ui.model
+
+import android.os.Parcelable
+import com.adyen.checkout.components.core.PaymentMethod
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+internal data class DropInPaymentMethodInformation(
+    val name: String
+) : Parcelable
+
+internal fun PaymentMethod.overrideInformation(information: DropInPaymentMethodInformation) {
```

Comment:
Minor: do you think that moving this closer to `overridePaymentMethodInformation` makes more sense? I also see the benefit of keeping it here since you have visibility over what's inside `DropInPaymentMethodInformation`, so not quite sure about this one.

Reply:
That was exactly how I thought about it, to keep the `overrideInformation` next to `DropInPaymentMethodInformation` for visibility and it would make it easy to modify them if a new field is added.